src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.Easy.cs

```diff
@@ -0,0 +1,144 @@
+            internal const int CURLOPT_PROXYPORT = CurlOptionLongBase + 59;
+            internal const int CURLOPT_POSTFIELDSIZE = CurlOptionLongBase + 60;
+            internal const int CURLOPT_MAXREDIRS = CurlOptionLongBase + 68;
+            internal const int CURLOPT_NOSIGNAL = CurlOptionLongBase + 99;
+            internal const int CURLOPT_PROXYTYPE = CurlOptionLongBase + 101;
+            internal const int CURLOPT_HTTPAUTH = CurlOptionLongBase + 107;
+            internal const int CURLOPT_PROTOCOLS = CurlOptionLongBase + 181;
+            internal const int CURLOPT_REDIR_PROTOCOLS = CurlOptionLongBase + 182;
+
+            internal const int CURLOPT_WRITEDATA = CurlOptionObjectPointBase + 1;
+            internal const int CURLOPT_URL = CurlOptionObjectPointBase + 2;
+            internal const int CURLOPT_PROXY = CurlOptionObjectPointBase + 4;
+            internal const int CURLOPT_PROXYUSERPWD = CurlOptionObjectPointBase + 6;
+            internal const int CURLOPT_READDATA = CurlOptionObjectPointBase + 9;
+            internal const int CURLOPT_COOKIE = CurlOptionObjectPointBase + 22;
+            internal const int CURLOPT_HTTPHEADER = CurlOptionObjectPointBase + 23;
+            internal const int CURLOPT_HEADERDATA = CurlOptionObjectPointBase + 29;
+            internal const int CURLOPT_CUSTOMREQUEST = CurlOptionObjectPointBase + 36;
+            internal const int CURLOPT_ACCEPT_ENCODING = CurlOptionObjectPointBase + 102;
+            internal const int CURLOPT_PRIVATE = CurlOptionObjectPointBase + 103;
+            internal const int CURLOPT_COPYPOSTFIELDS = CurlOptionObjectPointBase + 165;
+            internal const int CURLOPT_SEEKDATA = CurlOptionObjectPointBase + 168;
+            internal const int CURLOPT_USERNAME = CurlOptionObjectPointBase + 173;
+            internal const int CURLOPT_PASSWORD = CurlOptionObjectPointBase + 174;
+
+            internal const int CURLOPT_WRITEFUNCTION = CurlOptionFunctionPointBase + 11;
+            internal const int CURLOPT_READFUNCTION = CurlOptionFunctionPointBase + 12;
+            internal const int CURLOPT_HEADERFUNCTION = CurlOptionFunctionPointBase + 79;
+            internal const int CURLOPT_SSL_CTX_FUNCTION = CurlOptionFunctionPointBase + 108;
+            internal const int CURLOPT_SEEKFUNCTION = CurlOptionFunctionPointBase + 167;
+        }
+
+        // Class for constants defined for the enum CURLINFO in curl.h
+        internal static partial class CURLINFO
+        {
+            // Curl info are of the format <type base> + <n>
+            private const int CurlInfoStringBase = 0x100000;
+            private const int CurlInfoLongBase = 0x200000;
+
+            internal const int CURLINFO_PRIVATE = CurlInfoStringBase + 21;
+            internal const int CURLINFO_HTTPAUTH_AVAIL = CurlInfoLongBase + 23;
+        }
+
+        // AUTH related constants
+        internal static partial class CURLAUTH
```

Comment:
Yes, I'll make these all enums.  This has the (beneficial) side effect of changing all the "int" types that use them to the corresponding enum type.